src/components/Main.tsx

```diff
@@ -0,0 +1,89 @@
+import cn from 'classnames';
+import React from 'react';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  filteredTodos: Todo[],
+  isEditing: boolean,
+};
+
+export const Main: React.FC<Props> = ({ filteredTodos, isEditing }) => {
+  return (
```

Comment:
why not just returning right away?
```suggestion
export const Main: React.FC<Props> = ({ filteredTodos, isEditing }) => (
```